tesseract_motion_planners/examples/raster_example.cpp

```diff
@@ -112,14 +113,14 @@ int main(int /*argc*/, char** /*argv*/)
                                                   Eigen::Quaterniond(0, 0, -1.0, 0));
 
     // Define raster move instruction
-    MoveInstruction plan_c0(wp2, MoveInstructionType::LINEAR, "RASTER");
```

Comment:
It looks like all of he profiles were removed. Was there a reason behind this?

Reply:
None of them were ever actually added to the profile dictionary, so it was just pulling the default profile. That was the case in most of the examples actually. A lot of the changes I made in this PR were related to raising exceptions when requested profile names didn't have a corresponding profile entry in the dictionary.